src/_button.scss

```diff
@@ -0,0 +1,57 @@
+.button {
+  box-sizing: border-box;
+  padding: 12px 25px;
+  margin: 5px 5px 5px 0;
```

Comment:
5px, 12px, 25px can be moved to variables.

Reply:
I use 5px only 3 times, 12px only 1 time, 15 px only  1 time, 25 only 1 time, 30px only 2 times, 100px only 1 time... is It necessary to add this to variables as new, or I can use 
$font__size--headings-less: 15px;
$font__size--headings-least: 12px; for example? :(

Reply:
I don't think you should use $font__size variables for setting margin and padding.

As for adding variables for rarely used values -- we'll discuss it at the next lesson.
My opinion -- yes, you should create variables, but this is arguable -- you can leave everything as is at the moment.